Sources/Build/Buildable.swift

```diff
@@ -59,16 +59,24 @@ extension Module: Buildable {
     }
 }
 
-extension SwiftModule {
-    func pkgConfigArgs() throws -> [String] {
-        return try recursiveDependencies.flatMap { module -> [String] in
+extension ModuleProtocol {
+
+    func pkgConfigArgsCombined() throws -> [String] {
```

Comment:
Functions like this really need a documentation comment.

It isn't really clear what this function conceptually maps to. It is passing Xcc (escaped for `swiftc`) + Xld, but Xld _isn't_ escaped for `swiftc`. Abstractly, how can that be used correctly?

What probably happens is that Xld frequently includes arguments which are ok to pass straight to `swiftc`, but nothing is auditing that (instead, the user is just going to get weird errors from `swiftc` if a package tries to pass other linker flags).

I would probably try to rephrase these functions as "arguments for compiling with a GCC compatible driver (for C code)", "arguments for compiling with Swiftc" and "arguments for linking", and share code as appropriate.